core/core-model/src/main/java/com/hbc/pms/core/model/enums/AlarmStatusEnum.java

```diff
@@ -0,0 +1,15 @@
+package com.hbc.pms.core.model.enums;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.ToString;
+
+@AllArgsConstructor
+@ToString
+public enum AlarmStatusEnum {
+  TRIGGERED("TRIGGERED"),

```

Comment:
Is there any particular reason for duplicating the text "TRIGGERED" besides the name of the enum? 

For normal use cases where we need to get the string version of enum, I would use AlarmStatus.TRIGGERED.name()